src/crypto/crypto_util.h

```diff
@@ -696,24 +696,30 @@ template <typename T>
 class ArrayBufferOrViewContents {
  public:
   ArrayBufferOrViewContents() = default;
+  ArrayBufferOrViewContents(const ArrayBufferOrViewContents&) = delete;
+  void operator=(const ArrayBufferOrViewContents&) = delete;
 
   inline explicit ArrayBufferOrViewContents(v8::Local<v8::Value> buf) {
+    if (buf.IsEmpty()) {
```

Comment:
I understand why this exists, but it feels somewhat strange to explicitly allow passing in empty `Local`s.

Reply:
I agree. An alternative would be to allocate an empty `ArrayBuffer` and pass it into this function. None of the current callsites looked particularly hot, but I wasn't sure.

Do you think that would be acceptable wrt to performance?

(P.S. if your concern here is blocking, feel free to request changes on the PR.)